.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: node_js
+node_js:
+- 'stable'
+- '5.12.0'
+- '4.6.0'
+services: mongodb
+env:
+  global:
+    - export MONGO_URL=mongodb://localhost/appointment-reminders-node-test
```

Comment:
It does Not need the command `export` since this is parsed by Travis, just the variable named is Ok.
Like `- VARIABLE = value`.